static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -188,6 +188,12 @@ define([
             }
         });
 
+        if (window.location.hash.includes('devtools')) {
```

Comment:
looks like it? https://github.com/zloirock/core-js/blob/master/fn/string/includes.js